pvsite_datamodel/read/user.py

```diff
@@ -103,7 +103,11 @@ def get_all_site_groups(session: Session) -> List[SiteGroupSQL]:
     return site_groups
 
 
-def get_all_last_api_request(session: Session) -> List[APIRequestSQL]:
+def get_all_last_api_request(
+    session: Session,
+    include_in_url=None,
+    exclude_in_url=None,
```

Comment:
can you do `exclude_in_url: Optional[str] = None`, you my need to add a import, `from typing import Optional`

Could you add a sentence in hte function description too?